autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -117,6 +117,20 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
             Use `auto_stack` to maximize predictive accuracy; use `hyperparameter_tune` if you prefer to deploy just a single model rather than an ensemble.
         feature_prune : bool, default = False
             Whether or not to perform feature selection.
+        refit_full : bool, default = False
+            Whether to retrain all models on all of the data after the normal training procedure.
+            The time taken by this process is not enforced by `time_limits`.
+            If `True`, optimizes a model's inference time by collapsing bagged ensembles into a single model fit on all of the training data.
+            This process will typically result in a slight accuracy reduction and a large inference speedup.
+            The inference speedup will generally be between 10-200x faster than the original bagged ensemble model.
+            The added fit runtime of this function is generally 10% or less of the original fit runtime.
```

Comment:
Many of these statements are only relevant if user specified bagging should be used (via  `auto_stack`, `num_bagging_folds`...), so there should probably be a qualifying statement preceding them.

Reply:
Added